AWSiOSSDKv2.xcodeproj/project.pbxproj

```diff
@@ -12690,7 +12690,7 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				170C2D471EDFAE43008A2E52 /* AWSCognitoAuthTests.m in Sources */,
```

Comment:
Why was `AWSCognitoAuthTests` removed?

Reply:
That was a reference that included the file as part of the unit tests project. The file is still present, and linked. Of course, it's also empty, so it doesn't make a practical difference. :)